Cloudinary-pwa-kit/packages/pwa-cloudinary/app/components/cloudinary-image-gallery/index.jsx

```diff
@@ -58,9 +58,9 @@ const CloudinaryImageGallery = ({ size, cloudinaryImageGallery = {} }) => {
                         <>
                             <Box {...styles.heroImageGroup}>
                                 <AspectRatio {...styles.heroImage} ratio={1}>
-                                    <Img className={imageUrl?.isResponsive && 'cld-responsive'}
+                                    <Img className={imageUrl?.isResponsive ? 'cld-responsive' : ''}
                                         src={`${imageUrl.url.lastIndexOf('?') > -1 ? imageUrl.url.substring(0, imageUrl.url.lastIndexOf('?')) + cloudinary.CLD_TRACKING_PARAM : imageUrl.url + cloudinary.CLD_TRACKING_PARAM}`}
-                                        srcset={!imageUrl?.isResponsive && imageUrl?.srcset && updateTrackingParam(imageUrl?.srcset)}
+                                        srcSet={!imageUrl?.isResponsive && imageUrl?.srcset && updateTrackingParam(imageUrl?.srcset)}
```

Comment:
@asad-rafter If the conditions are not met, we will get srcset=false or sizes=false, which are out-of-spec. Please fix in all files

Reply:
Hi @yuval-cloudinary we have verified that if the conditions are not met, it's not setting it false. It doesn't include srcSet.